Yoast/wordpress-seo
# P1-1007 - Setup Jest

## Context


* Setup jest to test our code

## Summary


This PR can be summarized in the following changelog entry:

* Setup jest to test our code

## Relevant technical choices:

* Place the tests inside of the applicable folders (ducky pattern; [ref](https://www.freecodecamp.org/news/scaling-your-redux-app-with-ducks-6115955638be/#enter-re-ducks))

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

*


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

* `cd` into the project folder
* run `yarn test`
* Verify that the (config slice)test suite passes

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes P1-1007
